src/TwitterDB.js

```diff
@@ -0,0 +1,462 @@
+var SQLite3 = require('sqlite3').verbose();
+var log = require('npmlog');
+
+
+/**
+ * TwitterDB - Stores data for specific users and data not specific to rooms.
+ * @class
+ * @param  {string} filepath Location of the SQLite database file.
+ */
+var TwitterDB = function(filepath){
+  this.db = new SQLite3.Database(filepath,(err) => {
+    if(err){
+      log.error("TwitDB","Error opening database, %s");
```

Comment:
This is one of those rare cases where you probably *do* want to crash the process if this fails.